webpack.config.js

```diff
@@ -54,6 +54,7 @@ const commonSettings = {
         CLOSED_LOCATIONS: JSON.stringify(process.env.CLOSED_LOCATIONS),
         RECAP_CLOSED_LOCATIONS: JSON.stringify(process.env.RECAP_CLOSED_LOCATIONS),
         NON_RECAP_CLOSED_LOCATIONS: JSON.stringify(process.env.NON_RECAP_CLOSED_LOCATIONS),
+        DISPLAY_TITLE: JSON.stringify(process.env.DISPLAY_TITLE),
```

Comment:
Meant to push these two changes to #1647. Hopefully these two PRs can get reviewed and merged at a similar time.